OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs

```diff
@@ -30,7 +30,7 @@ public class AddressDto
 
     public double Longitude { get; set; }
 
-    public long CodeficatorId { get; set; }
+    public long CodeficatorId { get; set; } = 31737;
```

Comment:
I think to move this value to \OutOfSchool.Common\Constants.cs will be better

Reply:
done